src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VariantMongoWriter.java

```diff
@@ -118,19 +116,14 @@ private void createIndexes() {
                 new BasicDBObject(MongoDBHelper.BACKGROUND_INDEX, true));
 
         mongoOperations.getCollection(collection).createIndex(
-                new BasicDBObject(ANNOTATION_XREF_ID_FIELD, 1),
```

Comment:
Looking at the code diff, there were 2 indexes before (xref and SO) but just one after (SO). The xref index is still necessary.